cmd/fanlin/main.go

```diff
@@ -114,5 +114,8 @@ func main() {
 	http.Handle("/", h)
 	http.HandleFunc("/healthCheck", handler.HealthCheckHandler)
 
+	metricsHandler := handler.MakeMetricsHandler(conf, log.New(os.Stderr, "", log.LstdFlags))
+	http.HandleFunc("/metrics", metricsHandler.ServeHTTP)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I think the "enable" option is required. Because I am worried that metrics could be used by attackers.

Reply:
I added a directive to the configuration file to be able to set whether the metrics endpoint enabled or disabled. 18fd13f